src/ripple/app/tests/Flow_test.cpp

```diff
@@ -814,7 +814,7 @@ struct Flow_test : public beast::unit_test::suite
         {
             // First pass through a strand redeems, second pass issues, through an offer
             // limiting step is not an endpoint
-            Env env (*this, features(featureFlowV2));
+            Env env (*this, features(featureFlowV2), features(featureOwnerPaysFee));
```

Comment:
Just a clarification. The two modes would be "owner pays" and "sender pays", not "recipient pays". There are other tests that exercise "sender pays", just not every test (you do have a point, of course).